doc/dev/crimson/crimson.rst

```diff
@@ -10,14 +10,24 @@ be backward compatible with classic OSD.
 
 .. highlight:: console
 
+Obtaining Crimson
+=================
+
+The Crimson's source code is a part of the Ceph repository. To clone it::
+
+  $ git clone https://github.com/ceph/ceph.git
+  $ cd ceph
+
 Building Crimson
 ================
 
 Crimson is not enabled by default. To enable it::
 
   $ WITH_SEASTAR=true ./install-deps.sh
-  $ mkdir build && cd build
-  $ cmake -DWITH_SEASTAR=ON ..
+  $ ./do_cmake.sh -DWITH_SEASTAR=ON
+  $ cd build
+  $ ninja
```

Comment:
personally, i am not in favor of replicating the document which also applies to Ceph in general.

Reply:
I wasn't too. I changed my mind when I had to provide a step-by-step instruction to non-developers. Interleaving two docs brought huge amount of confusion :-(.